src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -307,6 +318,17 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
         return fields;
     }
 
+    /*******************************************************************************************************
+    * @description checks for read, create, and edit FLS for a given field
+    * @param dfr DescribeFieldResult of the field to check
+    * @return void, throws AuraHandledException to display UI error
+    */
+    private static void checkFLS(DescribeFieldResult dfr) {
+        if (!dfr.isAccessible() || !dfr.isCreateable() || !dfr.isUpdateable()) {
```

Comment:
Is there ever a case where isUpdateable() is true and either isAccessible() or isCreateable() is false? I would think those checks would be redundant, but let me know if that's not the case.

Reply:
They could have Edit but not Create, so we need to check for both of those, but I think you're right that isAccessible (View) is redundant. I'll take it out.